src/mjan.c

```diff
@@ -300,18 +329,95 @@ void DebugMenu(void) {}
 /*********** BEGIN VARIANT FUNCTIONS ***********/
 
 // as of this version, only standard exists
-int NumberOfOptions(void) {
-    return 1;
+int NumberOfOptions() {
+	return 2*3;
 }
-int getOption(void) {
-  return 0;
+
+int getOption() {
```

Comment:
i like what youre doing here. what about a switch that maps our variants to a range, and the multiplies by `-1` based on `gStandardGame`? might shorten `GetVarString`...